plugins/inputs/kafka_consumer/kafka_consumer_test.go

```diff
@@ -558,6 +560,14 @@ func TestKafkaRoundTripIntegration(t *testing.T) {
 				MaxUndeliveredMessages: 1,
 				ConnectionStrategy:     tt.connectionStrategy,
 			}
+			if tt.name == "topic regexp" {
+				t.Logf("rt: setting topic regexp and refresh")
+				input.Topics = nil
+				input.TopicRegexps = []string{topicRegexp}
+				timeInterval, _ := time.ParseDuration("5s")
+				interval := config.Duration(timeInterval)
+				input.TopicRefreshInterval = interval
+			}
```

Comment:
How about adding those parameters to the test-structure instead of doing this fragile iffery?

Reply:
Done.